libs/resource_partitioner/examples/CMakeLists.txt

```diff
@@ -25,19 +26,14 @@ if(HPX_WITH_MAX_CPU_COUNT AND HPX_WITH_SHARED_PRIORITY_SCHEDULER)
     oversubscribing_resource_partitioner
     simple_resource_partitioner
   )
+endif()
 
-  if (HPX_WITH_CXX14_RETURN_TYPE_DEDUCTION)
```

Comment:
Should this no longer be guarded by the above `if()`?

Reply:
No, with the rationale that if we require C++14 support we don't check for it unnecessarily. I'd only add checks for C++11 and 14 features that are not supported by compilers that we really care about (e.g. atomics need some special care).

Reply:
Further above, sorry. These commands were previously guarded by `if(HPX_WITH_MAX_CPU_COUNT AND HPX_WITH_SHARED_PRIORITY_SCHEDULER)`.

Reply:
Grr, yep. Fixed now.